server/svix-server/Cargo.toml

```diff
@@ -46,7 +46,7 @@ opentelemetry-otlp = { version = "0.10" }
 validator = { version = "0.14.0", features = ["derive"] }
 jwt-simple = "0.10.8"
 ed25519-compact = "1.0.11"
-chrono = { version="0.4.19", features = ["serde"] }
+chrono = { version="0.4.26", features = ["serde"] }
```

Comment:
The CVE is against certain versions of `time`, and chrono have been _taking a lot of time_ to decide if they want to make a breaking change in `0.4.x` to fix it, or wait for the `0.5.x` release line to start.

https://github.com/chronotope/chrono/issues/1073

Reply:
If you disable the `oldtime` feature (which is on by default for niche backwards compat cases) then it will pull in a newer version of the `time` crate. See: https://docs.rs/chrono/latest/chrono/#duration

Reply:
Yeah, I tried that but... I think there's something funky in cargo's features assessment where the offending `time` crate still winds up in the lockfile, and we still get dinged by dependabot.

I think it might have been because of other transitive deps on chrono, from things like seaorm.

Reply:
Ah, good point. It looks like our ksuid library might be the culprit:
![image](https://github.com/svix/svix-webhooks/assets/138695619/cfca8ecd-d85c-4ce5-92d3-5486b4501a1c)

Reply:
If the recent sea_orm update removed the issue on that side, we can probably fixup ksuid (assuming it's the last dependant in the graph) 🤔